docs/api.md

```diff
@@ -647,18 +648,6 @@ Failing requests may be caused
 by the following errors
 (this is not an exhaustive list):
 
-* `code: 400, errno: 125`:
-  The request was blocked for security reasons
-
-* `code: 400, errno: 142`:
-  Sign in with this email type is not currently supported
-
-* `code: 400, errno: 149`:
-  This email can not currently be used to login
-
-* `code: 400, errno: 127`:
-  Invalid unblock code
-
```

Comment:
Hrm, I guess the docs-generating script can no longer detect these errors, because they're thrown by shared helper functions rather than directly by the route.  @philbooth what do you think the right approach would be to correcting this?  It seems like important information that we don't want to lose.

Reply:
Yep, that script can only see into nested functions. Reliably seeing into other modules would require would require a fully-fledged JS interpreter, which is a whole different kettle of fish.

If you need to keep these errors right now, you could manually copy them to an earlier spot in the document, between the `<!--begin-route-post-accountlogin-->` and `<!--end-route-post-accountlogin-->` tags. Not ideal and difficult to do coherently, as it splits the errors between two places.

In the medium-term, I can think of a couple of mitigating changes we could make to the script:

1. Add begin/end tags for documenting additional errors, to use in cases like this (easy, but depends on human input).

2. Parse the local tests for each endpoint and add whatever errors we find inside assertions therein (a little bit harder, and depends on us having good test coverage).